sql/moz-fx-data-shared-prod/subscription_platform_derived/apple_subscriptions_changelog_v1/checks.sql

```diff
@@ -0,0 +1,2 @@
+#fail
+{{ is_unique(["id"]) }}
```

Comment:
It would probably be good to also add these checks to Bigeye as the bqetl checks will get deprecated at some point (no clear timeline yet). Either add them via the Bigeye UI or a Bigconfig file in here: https://mozilla.github.io/bigquery-etl/reference/bigconfig/